vcpkg.json

```diff
@@ -16,7 +16,14 @@
     "directxmath",
     "directxtk",
     "rapidcsv",
-    "spdlog"
+    "spdlog",
+    "openvr"
```

Comment:
Is this unpinning the openvr version? We specifically locked to the version shipped with VR.

Reply:
Ah yeah it does unpin it to v2.5.1, I wasn't aware it's currently pinned to the version VR ships with. v1.0.10 isn't available on vcpkg--the nearest available version is v1.0.12. Would that work you think?

Reply:
Maybe use a custom port for the git commit we had?

Reply:
Good idea, added a custom port in the latest commit